### PR TITLE
Add linux installer, persistent content dir, documents dir

### PR DESCRIPTION
**Pull request by Rafalkufel**

### CODE REVIEW

File: package.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+[ -e package ] && rm -r package
+mkdir -p package/opt
+mkdir -p package/usr/share/applications
+mkdir -p package/usr/share/icons/hicolor/scalable/apps
+
+cp -r dist/alinka package/opt/alinka
```
Comment (w1stler) [inline package.sh:7]: That works with whitespace?

File: etc/buildspec_merge_linux.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    POETRY_VIRTUALENVS_CREATE: false
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.9
+    commands:
+      - apt update; apt install -y ruby libxkbcommon-x11-0 libegl1 libwayland-cursor0 libwayland-egl1 libgtk-3-0 libatk1.0-0
+      - export PATH=$PATH:/root/.gem/ruby/$(ruby --version | awk '{print $2}' | cut -d "." -f 1-2).0/bin
+      - echo $PATH
```
Comment (w1stler) [inline etc/buildspec_merge_linux.yml]: Probably not required.

File: etc/buildspec_merge_linux.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      - gem install fpm --user-install
+      - ruby --version
```
Comment (w1stler) [inline etc/buildspec_merge_linux.yml]: ditto

File: etc/buildspec_merge_linux.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      - find / -type f -name fpm
```
Comment (w1stler) [inline etc/buildspec_merge_linux.yml]: ditto

File: app/config/configuration.py [hunk 5]
```diff
@@ -5,10 +5,15 @@
...
-    DB_PATH = os.path.join(BASE_PATH, DB_FILE_NAME)
+    USER_HOME_DIR: str = os.path.expanduser("~")
+
+    DB_FILE_NAME: str = "alinka.db"
+    PERSISTENT_DATA_DIR: str = ".alinka"
+    DOCUMENT_DIR_NAME: str = "Alinka-dokumenty"
+
+    DOCUMENTS_PATH: str = os.path.join(USER_HOME_DIR, DOCUMENT_DIR_NAME)
+    PERSISTENT_DATA_PATH: str = os.path.join(USER_HOME_DIR, PERSISTENT_DATA_DIR)
+    DB_PATH = os.path.join(PERSISTENT_DATA_PATH, DB_FILE_NAME)
```
Comment (w1stler) [inline app/config/configuration.py]: If you are typing stuff, this one is not. :P

**Review by w1stler — COMMENTED** [review]
Initial questions, but we might have a chat about some things, as I haven't worked with installers etc.

File: package.sh (hunk 1 above)
Comment (Rafalkufel) [reply package.sh:7]: Yes. It seems to be more readable with whitespaces.

**Review by w1stler — APPROVED** [review]

**Pull request merged**